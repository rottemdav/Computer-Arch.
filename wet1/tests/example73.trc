4 7 5 1 global_history local_tables not_using_share
0x47c0 N 0xd075b05c
0x61a7bc8 T 0x6e24fef0
0x09554f70 N 0xce44f78
0xe26cbc T 0xd78e04
0xe242784 T 0x4c4
0x505c T 0xa68
0xc7efed4 T 0x3a76c
0x8fc34 N 0x500240
0x505c T 0xa68
0x1828 T 0xebd680
0xec7d8 T 0xd18
0x3e010 N 0x315c
0x141c019c N 0xf0b640
0xb94f154 N 0x15ba5d44
0x47c0 T 0xd075b05c
0x141c019c T 0xf0b640
0x61a7bc8 N 0x6e24fef0
0x141c019c T 0xf0b640
0xc585c N 0x5e5f9c0
0xec7d8 N 0xd18
0x141c019c T 0xf0b640
0x0b4 N 0xea218
0xb94f154 N 0x15ba5d44
0x3e010 N 0x315c
0x1828 T 0xebd680
0xe26cbc N 0xd78e04
0x8fc34 T 0x500240
0x141c019c N 0xf0b640
0x4394 T 0x76b4
0x0b4 T 0xea218
0xc7efed4 N 0x3a76c
0x47c0 T 0xd075b05c
0xe26cbc N 0xd78e04
0x61a7bc8 N 0x6e24fef0
0x61a7bc8 N 0x6e24fef0
0x430 T 0x68ffc0
0xe26cbc N 0xd78e04
0xc7efed4 T 0x3a76c
0xc585c N 0x5e5f9c0
0x4a38 T 0xee709544
0x61a7bc8 N 0x6e24fef0
0x505c N 0xa68
0x47c0 T 0xd075b05c
0x61a7bc8 T 0x6e24fef0
0xe242784 T 0x4c4
0x505c N 0xa68
0xc7efed4 N 0x3a76c
0x141c019c N 0xf0b640
0x4b1c N 0xb775e460
0x4b1c N 0xb775e460
0x4b1c T 0xb775e460
0x54177b4 T 0x810
0xb94f154 T 0x15ba5d44
0x09554f70 N 0xce44f78
0xe26cbc N 0xd78e04
0xc112c8 N 0x7044
0x4394 N 0x76b4
0x09554f70 N 0xce44f78
0xe26cbc N 0xd78e04
0x4a38 N 0xee709544
0xc585c T 0x5e5f9c0
0xe242784 T 0x4c4
0x4394 N 0x76b4
0xc7efed4 N 0x3a76c
0xc585c N 0x5e5f9c0
0x4394 N 0x76b4
0x3e010 T 0x315c
0x0b4 N 0xea218
0x38cd98 T 0x418f1a8
0xc7efed4 N 0x3a76c
0x38cd98 T 0x418f1a8
0x505c T 0xa68
0x0b4 T 0xea218