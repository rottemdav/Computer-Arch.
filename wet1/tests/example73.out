0x47c0 N 0x47c4
0x61a7bc8 N 0x61a7bcc
0x9554f70 N 0x9554f74
0xe26cbc N 0xe26cc0
0xe242784 N 0xe242788
0x505c N 0x5060
0xc7efed4 N 0xc7efed8
0x8fc34 N 0x8fc38
0x505c N 0x5060
0x1828 N 0x182c
0xec7d8 N 0xec7dc
0x3e010 N 0x3e014
0x141c019c N 0x141c01a0
0xb94f154 N 0xb94f158
0x47c0 N 0x47c4
0x141c019c N 0x141c01a0
0x61a7bc8 N 0x61a7bcc
0x141c019c N 0x141c01a0
0xc585c N 0xc5860
0xec7d8 N 0xec7dc
0x141c019c N 0x141c01a0
0xb4 N 0xb8
0xb94f154 N 0xb94f158
0x3e010 N 0x3e014
0x1828 N 0x182c
0xe26cbc N 0xe26cc0
0x8fc34 N 0x8fc38
0x141c019c N 0x141c01a0
0x4394 N 0x4398
0xb4 N 0xb8
0xc7efed4 N 0xc7efed8
0x47c0 N 0x47c4
0xe26cbc N 0xe26cc0
0x61a7bc8 N 0x61a7bcc
0x61a7bc8 N 0x61a7bcc
0x430 N 0x434
0xe26cbc N 0xe26cc0
0xc7efed4 N 0xc7efed8
0xc585c N 0xc5860
0x4a38 N 0x4a3c
0x61a7bc8 N 0x61a7bcc
0x505c N 0x5060
0x47c0 N 0x47c4
0x61a7bc8 N 0x61a7bcc
0xe242784 N 0xe242788
0x505c N 0x5060
0xc7efed4 N 0xc7efed8
0x141c019c N 0x141c01a0
0x4b1c N 0x4b20
0x4b1c N 0x4b20
0x4b1c N 0x4b20
0x54177b4 N 0x54177b8
0xb94f154 N 0xb94f158
0x9554f70 N 0x9554f74
0xe26cbc N 0xe26cc0
0xc112c8 N 0xc112cc
0x4394 N 0x4398
0x9554f70 N 0x9554f74
0xe26cbc N 0xe26cc0
0x4a38 N 0x4a3c
0xc585c N 0xc5860
0xe242784 N 0xe242788
0x4394 N 0x4398
0xc7efed4 N 0xc7efed8
0xc585c N 0xc5860
0x4394 N 0x4398
0x3e010 N 0x3e014
0xb4 N 0xb8
0x38cd98 N 0x38cd9c
0xc7efed4 N 0xc7efed8
0x38cd98 N 0x38cd9c
0x505c N 0x5060
0xb4 N 0xb8
flush_num: 33, br_num: 73, size: 1175b
